nav2_map_server/README.md

```diff
@@ -100,6 +100,22 @@ Then, one would invoke this process with the params file that contains the param
 $ process_with_multiple_map_servers __params:=combined_params.yaml
 ```
 
+**Attention** : map_server is now a lifecycle node and needs to be transitioned to the active state. There are several ways you can trigger that state transition. The easiest way from the command line is:
```

Comment:
Remove this, this is a well documented item

Reply:
With that gone, I think the flow of this documentation section changes.

Reply:
I had to read another issue to discover this. It does not seem to be documented in the nav2_map_server package. But if you insist, I will remove it.

Reply:
You should still keep the ros2 run line for calling the lifecycle manager util for transitioning up the server, so I think that will be abundantly clear from context just below this line that its a lifecycle server